src/main.cpp

```diff
@@ -3726,7 +3773,6 @@ bool ProcessNewBlock(CValidationState& state, const CChainParams& chainparams, c
       }
 
     LogPrint("thin", "Processing new block %s from peer %s (%d).\n", pblock->GetHash().ToString(), pfrom ? pfrom->addrName.c_str():"myself",pfrom ? pfrom->id: 0);
-    if (!IsInitialBlockDownload()) SendExpeditedBlock(*pblock,pfrom);
```

Comment:
oh i see, but maybe you could change it to   if (IsChainNearlySyncd())
instead?

On 12/09/2016 2:46 PM, gandrewstone wrote:
>
> In src/main.cpp
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79#discussion_r78461029>:
>
> > @@ -3726,7 +3773,6 @@ bool ProcessNewBlock(CValidationState& state, const CChainParams& chainparams, c
> >        }
> >  
> >      LogPrint("thin", "Processing new block %s from peer %s (%d).\n", pblock->GetHash().ToString(), pfrom ? pfrom->addrName.c_str():"myself",pfrom ? pfrom->id: 0);
> > -    if (!IsInitialBlockDownload()) SendExpeditedBlock(*pblock,pfrom);
>
> Unfortunately it needs to be here to catch blocks coming from miners
> and from the relay network -- basically from the JSON interface, IIRC.
> I don't like the double check either though. I'm going to put this
> back in and then we can talk about what to do in a different PR. That
> way we can get everything else in this PR merged.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79/files/efaaa2474cc017ebc1c84fa3748b50c293bff5c7#r78461029>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMRF0Nm5yCxl7NTybDOQ3J96oIw5aUJ8ks5qpcgkgaJpZM4JokbP>.
>